dp/buy_sell_stock.py

```diff
@@ -23,12 +23,13 @@
 def max_profit(prices):
     """
     input: [7, 1, 5, 3, 6, 4]
-    diff : [X,-6, 5,-2, 3,-2]
+    diff : [X,-6, 4,-2, 3,-2]
```

Comment:
nit is the short form for 'nit pick'. It denotes a change which is not _important_, but is technically _correct_

I can see space missing before the negative numbers. (the reason I commented)